src/Components/Carousel/Carousel.css

```diff
@@ -1,7 +1,8 @@
 .react-multi-carousel-list {
   width: 100%;
 }
-
+.react-multi-carousel-track {
+}
 #Landing__Products {
```

Comment:
**question:** Wird diese id irgendwo verwendet? Ich sehe in der Codebase nichts dazu. Falls ja: Wenn ich global über das Projekt suche, dann muss dort `Landing__Products` an der entsprechenden Stelle auftauchen, damit klar ist, dass dieses ruleset verwendet wird